Swift Sources/UIViewController+Promise.swift

```diff
@@ -19,6 +20,15 @@ class MFMailComposeViewControllerProxy: NSObject, MFMailComposeViewControllerDel
         }
         PMKRelease(self)
     }
+
+    func messageComposeViewController(controller: MFMessageComposeViewController!, didFinishWithResult result: MessageComposeResult) {
+        if result.value == MessageComposeResultFailed.value {
+            controller.reject(NSError(domain: PMKErrorDomain, code: 4, userInfo: [NSLocalizedDescriptionKey: "The user’s attempt to save or send the message was unsuccessful."]))
```

Comment:
I guess there isn't a 4 for Swift PMK? Still we should fix it with a new code for both.